### PR TITLE
Andreea last update

### PR DESCRIPTION
**Pull request by AndreeaCh**

### CODE REVIEW

File: resource-management-client.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (acentea) [inline resource-management-client.iml]: I think this file shouldn't be commited.

File: src/store/modules/principal.js [hunk 2]
```diff
@@ -103,6 +110,16 @@ const mutations = {
       Vue.set(unit, 'resources', [])
     }
   },
+  [M.OPEN_CONFIRMATION_DIALOG] (state, unitName) {
+    let unit = state.units.find(u => u.name === unitName);
+    if(unit){
+      state.activeUnit = unit;
+    }
+    state.confirmationDialogIsOpen = true;
```
Comment (acentea) [inline src/store/modules/principal.js]: I'm not sure . Shouldn't we set confirmationDialogIsOpen  on true only if we found an unit ? ( in the if above)

File: resource-management-client.iml (hunk 1 above)
Comment (AndreeaCh) [reply resource-management-client.iml]: true. I will remove it.

File: src/store/modules/principal.js (hunk 2 above)
Comment (AndreeaCh) [reply src/store/modules/principal.js]: mmm. yes. Copy-paste from the existing code though ... 🥇

**Pull request merged**